test/system/src/main/java/io/pravega/test/system/framework/RemoteSequential.java

```diff
@@ -60,6 +60,8 @@
             if (response.status() != CREATED.code()) {
                 throw new TestFrameworkException(TestFrameworkException.Type.ConnectionFailed, "Error while starting " +
                         "test " + testMethod);
+            } else {
```

Comment:
this code is executed only for tests running on a mesos setup.

Reply:
Ack